meta-iot2000-bsp/conf/layer.conf

```diff
@@ -9,7 +9,7 @@ BBFILE_COLLECTIONS += "iot2000-bsp"
 BBFILE_PATTERN_iot2000-bsp = "^${LAYERDIR}/"
 BBFILE_PRIORITY_iot2000-bsp = "6"
 
-LAYERSERIES_COMPAT_iot2000-bsp = "rocko sumo thud zeus"
+LAYERSERIES_COMPAT_iot2000-bsp = "rocko sumo thud zeus dunfell"
```

Comment:
I wonder if we are really still backward-compatible... Anyway, not a blocker for this commit.

Reply:
Fixed.